modules/programs/vscode.nix

```diff
@@ -59,6 +62,41 @@ in
         '';
       };
 
+      keybindings = mkOption {
+        type = types.listOf (types.submodule {
+          key = mkOption {
+            type = types.str;
```

Comment:
I think you need to add the `options` here, so it becomes:
```
type = types.listOf ( types.submodule {
  options = {
    key = mkOption {
      ...
```

Reply:
Ah yeah, thanks.